src/ex1/script.js

```diff
@@ -0,0 +1,91 @@
+let addTaskBtn = document.querySelector(".submitBtn")
+let inputText = document.querySelector(".taskText")
+
+//Handle Image
+function addImg() {
+    const img = document.createElement("img");
+    img.id = "noTaskImg"
+    img.src = "https://memegenerator.net/img/instances/43020609.jpg";
+    const src = document.getElementById("div");
+    src.appendChild(img);
```

Comment:
A possible way to simplify this behavior is - instead of creating and destroying the img element over and over again
is to show and hide it using CSS. It will also prevent the funny bug from earlier